docker_xylem/service.py

```diff
@@ -143,7 +143,8 @@ def get_volume(self, request, data):
             return {
                 'Volume': {
                     'Name': name,
-                    'Mountpoint': self.current[name]
+                    'Mountpoint': self.current[name],
+                    'Status': {}
```

Comment:
It's generally a good idea to add trailing commas in list/dict literals, otherwise it's easy to add a new item at the end and forget to put a comma on the previous line.

The existing code doesn't have trailing commas, but this is an opportunity to improve it in a small but meaningful way. :-)